app/gather/settings.py

```diff
@@ -69,6 +69,9 @@
     if key.startswith(AETHER_PREFIX)
 ]
 
+ES_CONSUMER_URL = os.environ.get('ES_CONSUMER_URL')
```

Comment:
Can you include docs for `ES_CONSUMER_URL` environment variable? 📚

Reply:
:writing_hand:

Reply:
@lordmallam Done!

Reply:
Actually meant in `Read Me` with other environment variables. But I guess this is fine for now..

Reply:
:joy: 

You are right, but we can add it in the README once we finish the feature :thinking:

Reply:
@lordmallam in the right place! :nerd_face: